ios/Classes/ApptentiveFlutterPlugin.m

```diff
@@ -361,4 +397,9 @@ - (BOOL)isRegistered {
   return Apptentive.shared.apptentiveKey != nil && Apptentive.shared.apptentiveSignature != nil;
 }
 
+// Save the device token after registering for remote notifications
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
+    _device_token = deviceToken;
```

Comment:
You'll probably want to do `self.device_token` here. That calls the automatically-generated setter for `device_token` rather than just updating the pointer to point to the object. The setter (among other things) will make sure the reference count is incremented so the value doesn't get deallocated before you want it to.